README.md

```diff
@@ -321,43 +321,26 @@ Fixed versions:
 
 | Docker tag | Dockerfile            |
 | ---------- | --------------------- |
-| 1.10.0     | dockerfile/alpine_3.5 |
-| 1.10.1     | dockerfile/alpine_3.5 |
-| 1.10.2     | dockerfile/alpine_3.5 |
-| 1.10.3     | dockerfile/alpine_3.5 |
-| 1.10.4     | dockerfile/alpine_3.9 |
-| 1.10.5     | dockerfile/alpine_3.9 |
-| 1.10.6     | dockerfile/alpine_3.9 |
-| 1.10.7     | dockerfile/alpine_3.9 |
-| 2.1.0      | dockerfile/alpine_3.5 |
-| 2.1.1      | dockerfile/alpine_3.5 |
-| 2.1.2      | dockerfile/alpine_3.5 |
+| 1.10.[0-3] | dockerfile/alpine_3.5 |
```

Comment:
To be honest, I'm not sure about this change. Regex-free text is easier to read.

Can we ask someone else, how it looks better?

Reply:
Or we can split the long table to several shorter ones.

Reply:
I beg to disagree - those regexps make this table less verbose and easier to parse.

Reply:
Though we could put the same infomarmation as ranges, i.e.
`1.10.0 .. 1.10.3`

Reply:
Okay so. Ranges looks better for me. Anyway, Alexander, please pick up any choice you consider as the best.